interestingness/range.py

```diff
@@ -32,6 +32,11 @@
 path1 = os.path.abspath(os.path.join(path0, os.pardir, 'util'))
 sys.path.append(path1)
 
+try:
+    xrange
+except NameError:
+    xrange = range
+
```

Comment:
Why not just use range always? This is a repetition counter, so it is not going to be millions (and, on a modern machine, it still wouldn't be a problem if it was millions).